conftest.py

```diff
@@ -75,6 +75,13 @@ def pytest_addoption(parser):
              "4KiB blocksize to be formatted and used in storage tests. "
              "Set it to 'auto' to let the fixtures auto-detect available disks."
     )
+    parser.addoption(
+        "--sr-type",
```

Comment:
This is another added parameter to control which tests should be executed or not, which adds a layer between test definitions themselves and job definitions (jobs.py). I see a lot of PRs adding parameters, and this looks like added complexity for the ones running the tests.

How do you express "all storage types except largeblock and linstor" with this? (see job definitions in jobs.py that rely on this)

Reply:
This only include ext and lvm since those were identical. I agree it's not optimal, I'm looking into a way that will involve giving a path to pytest.

Reply:
Couldn't we use test class inheritance and still keep individual test files?

Reply:
I'm gonna try new things for this PR, gonna see if we can do something better with inheritance or other mechanisms.